### PR TITLE
feat(multiSelect): add support for exclusive options

### PR DESCRIPTION
**Pull request by maximilianoforlenza**
refactor(multiselect): add exclusive option

### CODE REVIEW

**Comment by maximilianoforlenza** [comment]
listo

File: src/components/MultiSelect/index.js [hunk 1]
```diff
@@ -18,14 +19,29 @@ const getSelectedValues = (optionValue, answer) => {
     return answerValue ? filter(answer, value => value !== optionValue) : concat(answer, optionValue);
```
Comment (navarroaxel) [inline src/components/MultiSelect/index.js]: rename `isAnswerIncluded`

File: src/components/MultiSelect/index.js [hunk 2]
```diff
@@ -18,14 +19,29 @@ const getSelectedValues = (optionValue, answer) => {
...
+    if (option.exclusive) {
+        answers = filter(
+            getSelectedValues(option.value, answer),
+            value => includes(getExclusiveOptions(question.options), value)
+        );
+    } else {
+        answers = filter(
+            getSelectedValues(option.value, answer),
+            value => !includes(getExclusiveOptions(question.options), value)
+        );
```
Comment (navarroaxel) [inline src/components/MultiSelect/index.js]:
```js
const exclusiveValues = getExclusiveOptionValues(question.options);
answers = filter(getSelectedValues(option.value, answer), value => !includes(exclusiveValues, value));
```

File: src/components/MultiSelect/index.js [hunk 3]
```diff
@@ -18,14 +19,29 @@ const getSelectedValues = (optionValue, answer) => {
     return answerValue ? filter(answer, value => value !== optionValue) : concat(answer, optionValue);
 };
 
+const getExclusiveOptions = options => map(filter(options, option => option.exclusive), option => option.value);
+
+const handleAnswer = (question, option, answer, onChange) => {
```
Comment (navarroaxel) [inline src/components/MultiSelect/index.js]: rename `handleChangeAnswer`

**Pull request merged**